src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/ErrorCodes.kt

```diff
@@ -1,33 +1,72 @@
 package uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config
 
+import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.magiclink.MAX_EMAIL_LENGTH
 import java.time.Duration
 import java.time.Instant
 import java.time.ZoneId
 import java.time.format.DateTimeFormatter
 
-sealed class ErrorCode(val code: String, val userMessage: String)
+@Schema(subTypes = [StandardErrorCodes::class, MagicLinkRequestErrorCodes::class, CheckBarcodeErrorCodes::class])
```

Comment:
Using subTypes instead of oneOf means that all subclasses are not included in the Schema for ErrorCode - this was confusing because not all subclasses are relevant to each endpoint.